NExtends/Primitives/Generics.extensions.cs

```diff
@@ -77,19 +75,6 @@ public static void RemoveRange<T>(this ICollection<T> collection, IEnumerable<T>
 			}
 		}
 
-		//Pour pouvoir écrire dans une NameValueCollection, il faut la convertir en Dictionnary
-		public static Dictionary<string, string> ToDictionary(this NameValueCollection collection)
-		{
-			Dictionary<string, string> dico = new Dictionary<string, string>();
-
-			foreach (string key in collection.AllKeys)
-			{
-				dico.Add(key, collection[key]);
-			}
-
-			return dico;
-		}
-
 		public enum SortOrder { Ascending, Descending };
 
 		public static IOrderedEnumerable<T> OrderByEnum<T>(this IQueryable<T> query, Func<T, object> orderDelegate, SortOrder sortOrder)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Consider using more general type 'System.Collections.Generic.IEnumerable<T>' instead of 'System.Linq.IQueryable<T>'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3242)